pandas/tests/dtypes/test_common.py

```diff
@@ -392,49 +392,15 @@ def test_is_not_unsigned_integer_dtype(dtype):
     assert not com.is_unsigned_integer_dtype(dtype)
 
 
-@pytest.mark.parametrize(
-    "dtype",
-    [
-        "int_",
-        "int8",
-        "Int8",
-        "int16",
-        "Int16",
-        "int32",
-        "Int32",
-        "int64",
-        "Int64",
-        "uint8",
-        "uint16",
-        "uint32",
-        "uint64",
-        "float_",
-        "float16",
-        "float32",
-        "Float32",
-        "float64",
-        "Float64",
-    ],
-)
-def test_all_type_comparison_with_true_result(dtype):
+def test_all_type_comparison_with_true_result(any_real_numpy_dtype):
```

Comment:
Could you also add the numeric ea dtypes?

Reply:
@phofl yes, i will add but my code doesnt pass all the tests because of some other files. What can i do for that?

=========================== short test summary info ============================
FAILED pandas/tests/arrays/floating/test_arithmetic.py::test_unary_float_operators[Float32-source0-neg_target0-abs_target0]
FAILED pandas/tests/arrays/floating/test_arithmetic.py::test_unary_float_operators[Float32-source1-neg_target1-abs_target1]
FAILED pandas/tests/arrays/floating/test_arithmetic.py::test_unary_float_operators[Float32-source2-neg_target2-abs_target2]
FAILED pandas/tests/arrays/floating/test_arithmetic.py::test_unary_float_operators[Float64-source0-neg_target0-abs_target0]
FAILED pandas/tests/arrays/floating/test_arithmetic.py::test_unary_float_operators[Float64-source1-neg_target1-abs_target1]
FAILED pandas/tests/arrays/floating/test_arithmetic.py::test_unary_float_operators[Float64-source2-neg_target2-abs_target2]
FAILED pandas/tests/arrays/floating/test_astype.py::test_astype_copy - Future...
FAILED pandas/tests/arrays/floating/test_comparison.py::TestComparisonOps::test_no_shared_mask[Float32Dtype]
FAILED pandas/tests/arrays/floating/test_comparison.py::TestComparisonOps::test_no_shared_mask[Float64Dtype]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int8-source0-neg_target0-abs_target0]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int8-source1-neg_target1-abs_target1]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int8-source2-neg_target2-abs_target2]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int16-source0-neg_target0-abs_target0]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int16-source1-neg_target1-abs_target1]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int16-source2-neg_target2-abs_target2]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int32-source0-neg_target0-abs_target0]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int32-source1-neg_target1-abs_target1]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int32-source2-neg_target2-abs_target2]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int64-source0-neg_target0-abs_target0]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int64-source1-neg_target1-abs_target1]
FAILED pandas/tests/arrays/integer/test_arithmetic.py::test_unary_int_operators[Int64-source2-neg_target2-abs_target2]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[Int8Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[Int16Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[Int32Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[Int64Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[UInt8Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[UInt16Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[UInt32Dtype]
FAILED pandas/tests/arrays/integer/test_comparison.py::TestComparisonOps::test_no_shared_mask[UInt64Dtype]
FAILED pandas/tests/arrays/integer/test_dtypes.py::test_astype_copy - FutureW...
= 30 failed, 122283 passed, 18661 skipped, 870 xfailed, 1 xpassed in 1386.44s (0:23:06) =

Reply:
@jreback @phofl My latest commit has passed all the tests, do i need to do anything else before merge?